src/ResourceManager/DataFactories/AzureRM.DataFactoryV2.psd1

```diff
@@ -75,6 +75,7 @@ FunctionsToExport = @()
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
 CmdletsToExport = 
                'Set-AzureRmDataFactoryV2',
+               'Update-AzureRmDataFactoryV2',
```

Comment:
Please update the change log with information about these two new cmdlets

Reply:
Done